static/js/stream_ui_updates.js

```diff
@@ -131,7 +131,7 @@ exports.update_stream_privacy_type_icon = function (sub) {
     }
 };
 
-exports.update_stream_privacy_type_text = function (sub) {
+exports.update_stream_subscription_type_text = function (sub) {
```

Comment:
renamed this function as both `privacy` and `stream_post_policy` values gets updated.

Reply:
Great!